doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -20,20 +20,148 @@
+| `req.gen.avl.01` | Availability | The Architecture **must** provide High Availability for Kubernetes components. |
+| `req.gen.ref.01` | Model | The Architecture **must** support the Reference Model defined profiles (instance types: Basic, Network Intensive, Compute Intensive). |
+
+
+<p align="center"><b>Table 2-1:</b> Kubernetes Architecture: General Requirements</p>
+
+<a name="2.3.2"></a>
+### 2.3.2 Infrastructure
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.inf.com.01` | Compute | The Architecture **must** provide compute resources for Pods.  |
+| `req.inf.com.02` | Compute | The Architecture **should** include industry standard hardware management systems at both HW device level (embedded) and HW platform level (external to device). |
+| `req.inf.com.03` | Network | The Architecture **must** support Conatiner Runtime Interface (CRI). |
+| `req.inf.stg.01` | Storage | The Architecture **must**  support for Kubernetes Volumes for container storage.
+| `req.inf.stg.02` | Storage | The Architecture **may** provide shared Object storage as a service for Containers workload.
+| `req.inf.stg.03` | Storage | The Architecture **must** support Container Storage Interfaces (CSI).
+| `req.inf.stg.04` | Storage | The Architecture **may** support Software Defined Storage (SDS) that seamlessly supports shared block storage, object storage and flat files. |
+| `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable storage. |
+| `req.inf.ntw.01` | Network | The Architecture **must** support intra-pod communications such as between all containers within a pod on localhost |
+| `req.inf.ntw.02` | Network | The Architecture **must** support intra-node communications, such as between agents on a node and all pods on that node |
+| `req.inf.ntw.03` | Network | The Architecture **must** support inter-node communications without NAT, such as communications between pods on a node with all other pods on all nodes |
+| `req.inf.ntw.04` | Network | The Architecture **must** support low latency and high throughput traffic needs. |
+| `req.inf.ntw.05` | Network | The Architecture **should** support service function chaining. |
+| `req.inf.ntw.06` | Network | The Architecture **must** support network resiliency. |
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace the concepts of open networking and disaggregation using commodity networking hardware and disaggregated Network Operating Systems. |
+| `req.inf.ntw.07` | Network | The Architecture **should** embrace open-based standards and technologies. |
+| `req.inf.ntw.08` | Network | The Architecture **must** be capable of supporting highly available (Five 9’s or better) workloads. |
```

Comment:
| `req.inf.ntw.08` | Network | The Architecture **must** be fully redundant, and all its component should be highly available
@pgoyal01 as agreed

Reply:
@pgoyal01 @tomkivlin we need to add : 
| req.inf.ntw.09 | Network | The Architecture must support Service Mesh Interface (SMI)

Reply:
> 
> 
> @pgoyal01 @tomkivlin we need to add :
> | req.inf.ntw.09 | Network | The Architecture must support Service Mesh Interface (SMI)

I do not see SMI at the moment as a mature enough option to be included in the RA. Also it is more like a PaaS service than an [IC]aaS.

Reply:
@CsatariGergely Maybe, but as per my observation on CNF vendors, they start adopting SMI and they request it to exist in the infrastructure 
So I recommend to keep it 
https://smi-spec.io/

Reply:
I think there is a lot of talk around Service Mesh, however there is little conclusion around implementation (it's more like a new SMI appears every month). So I am not sure if there would be a point in including it in the RA without a clear understanding of what implementation (API) to go for.

Reply:
@ASawwaf @CsatariGergely @TamasZsiros Following changes:

| `req.inf.ntw.08` | Network | The Architecture **must** be fully redundant. |
|-----------|----------|----------------------------|

and
| `req.inf.ntw.15` | Network | The Architecture **should** support Service Mesh Interface (SMI). |
|-----------|----------|----------------------------|